### PR TITLE
Documentation for integrating Okta with the Swift SDK

### PR DESCRIPTION
**Pull request by turnipdabeets**

### CODE REVIEW

File: docs/swift-sdk/integrate-with-okta.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+Before calling [`Embedded.shared.authenticate`](#authorize-with-okta), we must [Authenticate With Okta](integrate-with-okta#authorize-with-okta)
+
+### Authorize With Okta
+
+ - Step 1: Okta URL
+
+To begin the authentication flow, start an `ASWebAuthenticationSession`, and load the OAuth2 authorization request URL provided by Okta.
+
+```swift
```
Comment (ismailmustafa) [inline docs/swift-sdk/integrate-with-okta.md:21]: I'm not sure why, but for some reason, syntax highlighting doesn't work with `swift` or `kotlin`. There's probably some setting we need to modify with docusaurus, but in the meantime, I've been defaulting to javascript for everything just to get any sort of syntax highlighting.

File: docs/swift-sdk/integrate-with-okta.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+ - Step 2: Invoke URL
+
+During the session completionHandler, a URL with the invoke URL scheme should be returned from Okta. When the webpage loads a URL, call `Embedded.shared.authenticate`.
+
+```swift
+let session = ASWebAuthenticationSession(
+    url: viewModel.oktaURL,
+    callbackURLScheme: viewModel.callbackScheme
+){ (url, error) in
+    Embedded.shared.authenticate(
```
Comment (ismailmustafa) [inline docs/swift-sdk/integrate-with-okta.md:40]: I would also use `Embedded.shared.isAuthenticateUrl`. I know we expect the next URL to be a BI URL, but we want to:
- keep making developers aware of this functionality everywhere in our documentation
- be a bit safer in the off chance that a developer's flow is more complex that it requires this routing up front.

Comment (turnipdabeets) [reply docs/swift-sdk/integrate-with-okta.md:41]: `Embedded.shared.authenticate` calls `isAuthenticateUrl` under the hood so if they pass a non BI URL they will get an invalidURL error. 
But we could use this as an opportunity to document `isAuthenticateUrl` and I guess it doesn't hurt to call that function twice.

**Comment by nmelo** [comment]
@ismailmustafa ready to merge?

**Comment by Will5** [comment]
Unfortunately [this](https://github.com/gobeyondidentity/developer-docs/pull/57/commits/0fb0a6ad61140d9652abd3198bd3bf82e5c92a63) commit broke the navigation to "Integrate With Okta"

File: docs/swift-sdk/integrate-with-okta.md [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+---
+
+This guide describes how to configure Beyond Identity using Okta as the OIDC authentication flow. 
+
+### Prerequisites
+
+ - [Integrate With Okta](../sso-integrations/integrate-with-okta)
+ - [Swift SDK Overview](overview)
+
+Before calling [`Embedded.shared.authenticate`](#authorize-with-okta), we must [Authenticate With Okta](integrate-with-okta#authorize-with-okta)
```
Comment (Will5) [inline docs/swift-sdk/integrate-with-okta.md:13]: I think this first link is supposed to be "overview#authentication"

File: docs/swift-sdk/integrate-with-okta.md [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+
+### Authorize With Okta
+
+ - Step 1: Okta URL
```
Comment (ismailmustafa) [inline docs/swift-sdk/integrate-with-okta.md:17]: Matching my review from Will's, let's make this "Okta Authorize URL"

File: docs/swift-sdk/integrate-with-okta.md [hunk 5]
```diff
@@ -0,0 +1,112 @@
+---
+title: Integrate With Okta
+sidebar_position: 2
+---
+
+This guide describes how to configure Beyond Identity using Okta as the OIDC authentication flow.
```
Comment (ismailmustafa) [inline docs/swift-sdk/integrate-with-okta.md:6]: "This guide describes how to configure Okta to delegate to Beyond Identity for authentication during an OAuth2 authorization flow."

**Review by ismailmustafa — CHANGES_REQUESTED** [review]

File: docs/sdks/swift-sdk/integrate-with-okta.md [hunk 6]
```diff
@@ -0,0 +1,112 @@
...
+### Prerequisites
+
+ - [Integrate With Okta](../../sso-integrations/integrate-with-okta)
+ - [Swift SDK Overview](overview)
+
+Before calling [`Embedded.shared.authenticate`](overview#authentication), we must [Authenticate With Okta](integrate-with-okta#authorize-with-okta)
+
+### Authorize With Okta
+
+ - Step 1: Okta Authorize URL
```
Comment (ismailmustafa) [inline docs/sdks/swift-sdk/integrate-with-okta.md:17]: Also noticed this on Will's documentation:

I realized there's actually a step missing here before step 1, configuring the authenticator config to:

- be set to type "Embedded"
- set the invoke_url to the app_scheme or universal url/App link of your application

You can also point to https://developer-docs-git-v1-beyondidentity.vercel.app/docs/v1/platform-overview/authenticator-config#embedded for more context on "Embedded" authenticator config.

Comment (turnipdabeets) [reply docs/sdks/swift-sdk/integrate-with-okta.md]: sure, added

**Review by ismailmustafa — APPROVED** [review]

**Pull request merged**